packages/account/src/Sections/Profile/PhoneVerification/__test__/confirm-phone-number.spec.tsx

```diff
@@ -40,34 +57,88 @@ describe('ConfirmPhoneNumber', () => {
         expect(screen.getByRole('button', { name: 'Get code via SMS' })).toBeInTheDocument();
         expect(screen.getByRole('button', { name: 'Get code via WhatsApp' })).toBeInTheDocument();
     });
-    it('should call requestOnWhatsApp when Whatsapp button is clicked', () => {
+
+    it('should render validatePhoneNumber function on user key pressed', () => {
```

Comment:
can we change the description to a more readable one
it('should call validatePhoneNumber when the user presses a key', () => {